vumi/message.py

```diff
@@ -219,6 +231,8 @@ def process_fields(self, fields):
         fields.setdefault('content', None)
         fields.setdefault('transport_metadata', {})
         fields.setdefault('group', None)
+        fields.setdefault('to_addr_type', self.AT_MSISDN)
+        fields.setdefault('from_addr_type', self.AT_MSISDN)
```

Comment:
I wonder if we should allow ``None`` to be a value and have it mean ``unspecified``?